packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -299,38 +305,42 @@ interface ActionsBarProps extends IContainerActions {
 function ActionsBar(props: ActionsBarProps): React.ReactElement {
 	const { isContainerConnected, tryConnect, forceDisconnect, closeContainer } = props;
 
+	
 	const connectButtonTooltipId = useId("connect-button-tooltip");
 	const disconnectButtonTooltipId = useId("disconnect-button-tooltip");
 	const disposeContainerButtonTooltipId = useId("dispose-container-button-tooltip");
 
```

Comment:
Just a thought for general improvement: If the container is closed and/or disposed, we probably want to disable both of these buttons, since they won't do anything going forward. Not really in the scope of this PR, so feel free to close without action (though we should file a task in that case).

Reply:
Changes made: 

1. When `devTools` first initialized (Both buttons are available).
<img width="628" alt="image" src="https://user-images.githubusercontent.com/111468570/232613365-a99a233c-8863-4a50-8de3-338fb572f8bf.png">


2. After clicking `Disconnect Container` (Both buttons still available, but changed from `disconnect` to `connect`). 
<img width="628" alt="image" src="https://user-images.githubusercontent.com/111468570/232613548-2854f9de-2664-46f4-a9fd-d369c3069968.png">


3. After clicking `Close Container` (Both buttons are disabled). 

<img width="628" alt="image" src="https://user-images.githubusercontent.com/111468570/232613766-f817823e-0a2b-4251-90d8-ce268d578ff6.png">